packages/edit-site/src/components/editor/index.js

```diff
@@ -258,8 +258,7 @@ export default function EditSiteEditor( { isPostsList = false } ) {
 										whileTap="tap"
 									>
 										<Button
-											// TODO: Switch to `true` (40px size) if possible
-											__next40pxDefaultSize={ false }
+											__next40pxDefaultSize
```

Comment:
This can be seen when you go to site editor --> styles, check for the left top icon to go back to dashboard or other page.

**Note:** This does not have impact because of some overridden styles.

| Before | After     |
| :-------- | :------- |
| <img width="361" alt="Editor-close-button-before" src="https://github.com/user-attachments/assets/af66239e-fefe-41c3-be30-e5de1b2f06c6"> | <img width="361" alt="Editor-close-button-after" src="https://github.com/user-attachments/assets/a0fac3d0-909d-471e-b6d3-9777a5ee37b9"> |